test/e2e-common.sh

```diff
@@ -17,16 +17,20 @@
 # This script provides helper methods to perform cluster actions.
 source $(dirname $0)/../vendor/knative.dev/test-infra/scripts/e2e-tests.sh
 
-# Latest serving release. This is intentionally hardcoded for now, but
-# will need the ability to test against the latest successful serving
-# CI runs in the future.
-readonly LATEST_SERVING_RELEASE_VERSION=$(git describe --match "v[0-9]*" --abbrev=0)
+# Latest serving operator release.
+readonly LATEST_SERVING_OPERATOR_RELEASE_VERSION=$(git tag | sort -V | tail -1)
+# Latest serving release. This can be different from LATEST_SERVING_OPERATOR_RELEASE_VERSION.
+LATEST_SERVING_RELEASE_VERSION="v0.12.1"
 # Istio version we test with
 readonly ISTIO_VERSION="1.4.2"
 # Test without Istio mesh enabled
 readonly ISTIO_MESH=0
 # Namespace used for tests
-readonly TEST_NAMESPACE="operator-tests"
+readonly TEST_NAMESPACE="knative-serving"
+
+OPERATOR_DIR=$(dirname $0)/..
```

Comment:
```suggestion
OPERATOR_DIR=$(cd $(dirname "$0")/.. && pwd)
```

`OPERATOR_DIR=$(dirname $0)/..` resolves to `./..`. 
Changing it with `OPERATOR_DIR=$(cd $(dirname "$0")/.. && pwd)` results in `OPERATOR_DIR` being e.g. `/home/ali/go/src/knative.dev/`